translations/base-en.yaml

```diff
@@ -1209,21 +1209,20 @@ tips:
     - You don't need to divide up items evenly for full efficiency.
     - Holding <b>SHIFT</b> will activate the belt planner, letting you place long lines of belts easily.
     - Cutters always cut vertically, regardless of their orientation.
-    - To get white mix all three colors.
     - The storage buffer prioritises the left output.
     - Invest time to build repeatable designs - it's worth it!
-    - Holding <b>CTRL</b> allows to place multiple buildings.
+    - Holding <b>SHIFT</b> lets you place multiple buildings in one click.
```

Comment:
"at a time" may be better than "in one click", since multiplace also lets you just repeatedly click instead of just drag